packages/ui/src/memberships/components/components.tsx

```diff
@@ -58,6 +59,8 @@ export const MemberPhoto = styled.div<MemberPhotoProps>`
   align-items: center;
   max-width: ${({ small }) => (small ? '24px' : '40px')};
   max-height: ${({ small }) => (small ? '24px' : '40px')};
+  max-width: ${({ big }) => (big ? '80px' : '40px')};
+  max-height: ${({ big }) => (big ? '80px' : '40px')};
```

Comment:
```suggestion
  ${({ big, small }) => {
    if (big) {
      return css`
        max-width: 80px;
        max-height: 80px;
      `
    } else if (small) {
      return css`
        max-width: 24px;
        max-height: 24px;
      `
    } else {
      return css`
        max-width: 40px;
        max-height: 40px;
      `
    }
  }}
```
This is a lot longer but easier too read IMO